packages/web/src/components/Header/styles.scss

```diff
@@ -1,4 +1,6 @@
+@import '../../assets/styles/theme';
```

Comment:
can be changed to @import 'assets/styles/theme';

Reply:
It doesn't work